test/core/end2end/tests/short_deadlines.c

```diff
@@ -0,0 +1,200 @@
+/*
+ *
+ * Copyright 2015, Google Inc.
```

Comment:
We aren't changing those things anymore I think. Perhaps for new files we are? Hmm